src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -986,6 +986,10 @@ enum FilterPlaceType {
 
     """Parking lots that contain spaces for cars"""
     CAR_PARK
+
+
+    """Stops or stations"""
```

Comment:
I think we should add more documentation here what this actually does and also it's relation to the STOP filter (and vice versa on the STOP filter).

Reply:
Ok I think agree with you, lets do it like that. We need to document this behaviour well here in the schema.

Reply:
Yes, good documentation is important because it's going to be confusing no matter what the semantics will be.